protocols/discord.py

```diff
@@ -528,6 +549,25 @@ def wrap_message(source, target, text):
         """
         return [text]
 
+    def _get_or_create_webhook(self, channel):
```

Comment:
Most functions that operate on channels operate on their name; so this should either be named channelobj or the function should take in just the name.